### DISCUSSION TITLE
How to run weblate locally?

### DISCUSSION THREAD

**Discussion by ashishnagar19** [WeblateOrg/weblate#9346] [Q&A]
I tried everything, which has been said in the documentation and everywhere else. But i still can't figure out how to run the weblate locally. Can someone please provide me proper steps (please don't mention documentation). I really wanted to contribute to this project but i cannot help as i am unable to set up locally. Please help. Thanks in advance.

**Comment by nijel** [comment]
The documentation is the right place to find this information. If something doesn't work as documented (for example because it was documented using older versions), it needs to be fixed.

Which of https://docs.weblate.org/en/latest/contributing/start.html did you try, and how did it fail?
